Suppress -Wundef warning in glibc-rh841653-16.patch.  Specific to
the previous backport, so no upstream patch.

diff --git a/elf/dl-tunables.c b/elf/dl-tunables.c
index b7c735891d66bac0..f86a6853d2f9b2e3 100644
--- a/elf/dl-tunables.c
+++ b/elf/dl-tunables.c
@@ -27,7 +27,7 @@ void
 internal_function
 _dl_process_tunable_env_entries (void)
 {
-#if HAVE_ELISION
+#ifdef HAVE_ELISION
   char **ep;
   const char *envname = { "RHEL_GLIBC_TUNABLES" };
 # define TUNABLE_ELISION 0
